turbo.json

```diff
@@ -36,6 +36,17 @@
     "clean": {
       "cache": false
     },
-    "serve": { "cache": false, "persistent": true }
+    "serve": {
+      "cache": false,
+      "persistent": true
+    },
+    "check-deduped-deps": {
+      "inputs": [
+        "package.json",
+        "apps/*/package.json",
```

Comment:
shouldn't this be 
`apps/*/package.json` --> `apps/**/package.json`
`packages/*/package.json` --> `packages/**/package.json`

Reply:
Do you have nested `package.json`s? Maybe?

Reply:
mmm, no, we just have 1 package.json per workspace.